chained-batch.js

```diff
@@ -1,25 +1,32 @@
 const util = require('util')
 const AbstractChainedBatch = require('abstract-leveldown').AbstractChainedBatch
+const binding = require('./binding')
 
 function ChainedBatch (db) {
   AbstractChainedBatch.call(this, db)
-  this.binding = db.binding.batch()
+  this.context = binding.batch_init(db.context)
```

Comment:
Here's how we do it, regardless of where we are. We use an `_init` method to get a `context`.